hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/AbstractScheduleHandler.java

```diff
@@ -232,8 +234,11 @@ protected boolean tryToExecuteSchedule(
             //     so that the child transaction has those as "valid" keys.
             final var assistant = new ScheduleVerificationAssistant(validSignatories, new HashSet<>());
             final TransactionBody childTransaction = HandlerUtility.childAsOrdinary(scheduleToExecute);
-            final ScheduleRecordBuilder recordBuilder =
-                    context.dispatchChildTransaction(childTransaction, ScheduleRecordBuilder.class, assistant);
+            final ScheduleRecordBuilder recordBuilder = context.dispatchChildTransaction(
+                    childTransaction,
+                    ScheduleRecordBuilder.class,
+                    assistant,
+                    requireNonNull(scheduleToExecute).payerAccountId());
```

Comment:
This isn't the best way to do this...  I can fix in my next PR, however.

Reply:
Reverted.